tests/conftest.py

```diff
@@ -378,3 +378,47 @@ def test_matrix_4() -> np.ndarray:
     # fmt: on
 
     return p
+
+
+from moscot.datasets import simulate_data
```

Comment:
This will cause an issue if `moscot` is not installed, you might need to defined these fixtures differently.